#       modified:   jni/com_android_bluetooth_btservice_AdapterService.cpp
#       modified:   src/com/android/bluetooth/btservice/AdapterService.java
diff --git a/packages/apps/Bluetooth/jni/com_android_bluetooth_btservice_AdapterService.cpp b/packages/apps/Bluetooth/jni/com_android_bluetooth_btservice_AdapterService.cpp
index 48e3eb3..4a9be7c 100644
--- a/packages/apps/Bluetooth/jni/com_android_bluetooth_btservice_AdapterService.cpp
+++ b/packages/apps/Bluetooth/jni/com_android_bluetooth_btservice_AdapterService.cpp
@@ -1083,12 +1083,6 @@ static jboolean configHciSnoopLogNative(JNIEnv* env, jobject obj, jboolean enabl
 
     jboolean result = JNI_FALSE;
 
-    if (!sBluetoothInterface) return result;
-
-    int ret = sBluetoothInterface->config_hci_snoop_log(enable);
-
-    result = (ret == BT_STATUS_SUCCESS) ? JNI_TRUE : JNI_FALSE;
-
     return result;
 }
 
diff --git a/packages/apps/Bluetooth/src/com/android/bluetooth/btservice/AdapterService.java b/packages/apps/Bluetooth/src/com/android/bluetooth/btservice/AdapterService.java
index 395f024..d507523 100644
--- a/packages/apps/Bluetooth/src/com/android/bluetooth/btservice/AdapterService.java
+++ b/packages/apps/Bluetooth/src/com/android/bluetooth/btservice/AdapterService.java
@@ -942,15 +942,7 @@ public class AdapterService extends Service {
         }
 
         public boolean configHciSnoopLog(boolean enable) {
-            if ((Binder.getCallingUid() != Process.SYSTEM_UID) &&
-                (!Utils.checkCaller())) {
-                Log.w(TAG,"configHciSnoopLog(): not allowed for non-active user");
                 return false;
-            }
-
-            AdapterService service = getService();
-            if (service == null) return false;
-            return service.configHciSnoopLog(enable);
         }
 
         public void registerCallback(IBluetoothCallback cb) {
@@ -1470,8 +1462,7 @@ public class AdapterService extends Service {
      }
 
     boolean configHciSnoopLog(boolean enable) {
-        enforceCallingOrSelfPermission(BLUETOOTH_PERM, "Need BLUETOOTH permission");
-        return configHciSnoopLogNative(enable);
+        return false;
     }
 
      void registerCallback(IBluetoothCallback cb) {
